temporal.yaml

```diff
@@ -42,6 +42,14 @@ subpackages:
           ldflags: -s -w
       - uses: strip
 
+  - name: temporal-compat
+    description: "Compat package for temporal"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
+          ln -s /usr/bin/temporal "${{targets.contextdir}}"/usr/local/bin
+      - uses: strip
```

Comment:
This is just a symlink. I don't think we need `uses: strip`.

Reply:
Oh yes, it was a copy pasta and it's happening at many places so I think we need to progressively fix that too, let me fix it here for now, thanks!

Reply:
fixed, thanks